.github/workflows/runTests.yml

```diff
@@ -42,5 +42,28 @@ jobs:
       with:
         tag: nightly
         overwrite: true
-        file: '${{ runner.os }}.zip'
+        file: '../${{ runner.os }}.zip'
         asset_name: '${{ runner.os }}.zip'
+  
+  cpp-deploy:
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    needs: [test]
+    steps:
+    - name: Checkout Source
+      uses: actions/checkout@v4
+      with:
+        ref: generated_source
+        path: rat_source
+    - name: Deploy Souce
+      run: |
+        wget https://github.com/RascalSoftware/RAT/releases/download/nightly/Linux.zip
+        unzip Linux.zip
+        rm -rf rat_source/*.c rat_source/*.cpp rat_source/*.h
```

Comment:
Have you accounted for the files: "classHandle.hpp"  "dylib.hpp"  "eventHelper.hpp" & "rtw_proj.tmw". They are included in cppDeploy but not deleted here. Is that ok?

Reply:
I will update this line for .hpp but remove rtw_proj.tmw from the Matlab